app/routes/goal_routes.py

```diff
@@ -1 +1,113 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response
+from app.models.goal import Goal
+from app.routes.route_utilities import validate_model
+from ..db import db
+from app.models.task import Task
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@goals_bp.post("")
+def create_goal():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return {"details": "Invalid data"}, 400
+
+    new_goal = Goal(title=request_body["title"])
```

Comment:
This is a good check. One way we could actually combine it with the creation of the new goal would be to use a try/except block.  This might look something like this: 

     try:
        new_goal = Goal.from_dict(request_body)

     except KeyError as error:
        response = {"details": f"Invalid data"}
        abort(make_response(response, 400))
        
 Notice that we've also made use of the Goal model's from_dict method here!